src/style.css

```diff
@@ -1,3 +1,166 @@
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --white: #fff;
+  --lightgrey: #f3f3f3;
+  --darkgrey: #060b35;
+  --grey: #616070;
+  --blue: #00acdc;
+}
+
+.card {
+  width: 200px;
+}
+
+.card__rectangle {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+
+  width: 198px;
+  flex-shrink: 0;
+
+  border-radius: 5px;
+  border: 1px solid var(--lightgrey);
+  background: var(--white);
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  flex-shrink: 0;
+  margin: 32px 19px 0;
+}
+
+.card__description {
+  color: var(--darkgrey);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 40px 16px 0;
+}
+
+.card__code {
+  color: var(--grey);
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 14px;
+  margin: 4px 82px 0 16px;
+}
+
+.card__price-bloc {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 24px 14px 0 16px; /*верхній був 56*/
+}
+
+.card__price-word {
+  display: inline-block;
+
+  color: var(--grey);
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+}
+
+.card__price-value {
+  display: inline-block;
+  color: var(--darkgrey);
+  text-align: right;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--blue);
+  width: 166px;
+  height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Just use padding